libs/scalanativelib/src/mill/scalanativelib/ScalaNativeModule.scala

```diff
@@ -17,6 +17,7 @@ import mill.scalanativelib.worker.{
 import mill.T
 import mill.constants.EnvVars
 import mill.scalanativelib.worker.api.ScalaNativeWorkerApi
+import upickle.implicits.namedTuples.default.given
```

Comment:
Is there possibly something that can be done to make this import visible by default?

Reply:
It would be nice, yes. I'm open for suggestions. IIUC, if we `export` it from the location where it is needed (e.g. `TestModule`), we risk getting it into scope multiple times. We could just export it from `mill`, then at least `import mill._` will get it automatically?

Reply:
I think exporting it from JsonFormatters (which ends up inherited by mill._) is probably the best thing to do. That's where we put our other upickle implicits anyway so it'll fit right in

Reply:
* https://github.com/com-lihaoyi/mill/pull/5258